core/domain/collection_domain.py

```diff
@@ -1112,6 +1112,25 @@ def get_node(self, exploration_id):
                 return node
         return None
 
+    def get_sorted_nodes(self):
+        """Returns collection nodes list in sorted order.
```

Comment:
It may be good to clarify in this documentation that this method is assuming the collection only contains nodes in a linear order.